index.html

```diff
@@ -46,6 +46,17 @@
 
       window.currentTeam = window.CONFIG.TEAMS.ONE_ACCOUNT;
     </script> 
+    <script type="text/javascript">
+    document.addEventListener("DOMContentLoaded", (event) => {
+        console.log("DOM fully loaded and parsed");
+        const params = window.location.search.substring(1).split("&").reduce((prev ,item) => {
```

Comment:
It does not select One Account as default when there is no `team` param in query 🤷‍♂️ - thus we would need to change bookmarks everywhere and we are to lazy for that..

![image](https://github.com/heureka/on-calls-pre-fill/assets/3725315/0bb47012-4c57-4704-8fe7-64f7a6e263c9)

And pls mention this `team` query param in README.MD so others can use this cool stuff 👍